src/sections/Blog/Blog-sidebar/index.js

```diff
@@ -9,6 +9,7 @@ import { HiOutlineChevronDown } from "@react-icons/all-files/hi/HiOutlineChevron
 // import { FaSearch } from "@react-icons/all-files/fa/FaSearch";
 
 const Discuss = "../../../assets/images/discuss/layer5-discuss-white.webp";
+const FiveandFriendsAdventures = "../../../assets/images/blog/five-and-friends.png";
```

Comment:
Could we just use the already present one instead of adding another new image of the `Five and Friends`?

What do you think @toth2000?

Reply:
@Mohith234 are you referring to the image `src\assets\images\adventure-five\layer_five.png`? 
Both the images have different texts on it.

Reply:
Yes, IMO that text represented a bit more intuitive as a CTA. But yeah I saw the mockup has this image you've used so it's ok 👍 @toth2000

Reply:
@ritiksaxena124 Which one do you think is good? Any suggestions?

Reply:
![journey 2 (2)](https://github.com/layer5io/layer5/assets/51539117/f2c86783-99cc-40ee-8da1-856cd6ecb3e5)


we can use this image. I have updated the text

Reply:
@ritiksaxena124 @Mohith234 updated the image.